lib/workers/pr/index.spec.ts

```diff
@@ -643,5 +643,17 @@ describe('workers/pr', () => {
         gitLabAutomerge: true,
       });
     });
+
+    it('should create a PR with set of labels and mergeable addLabels', async () => {
+      config.labels = ['deps', 'renovate'];
+      config.addLabels = ['deps', 'js'];
```

Comment:
```suggestion
```

Reply:
Ooops, I forgot to remove these lines. Thanks :)

Reply:
It seems if no global config object is mofidied this tests fails, not sure why.
I think I will probably go back and remove the new config object to use the global mutable one 🤷‍♂️